adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1155,46 +1160,47 @@ public void startActivityForResult(Intent intent, int requestCode) {
 
     /**
      * Util method to merge
+     *
      * @param claims input claims passed on acquireToken call
      * @return merged claims with capabilities
      * @throws JSONException if input claims is an invalid JSON
-     *
-     * Sample input claim :
-     *      {
-                "userinfo":
-                {
-                    "given_name": {"essential": true},
-                    "email": {"essential": true},
-                },
-                "id_token":
-                {
-                    "auth_time": {"essential": true},
-                }
-            }
-
-     Sample capabilities list : [CP1, CP2 CP3]
-
-     Output merged claims :
-        {
-            "userinfo": {
-                "given_name": {
-                    "essential": true
-                },
-                "email": {
-                    "essential": true
-                }
-            },
-            "id_token": {
-                "auth_time": {
-                    "essential": true
-                }
-            },
-            "access_token": {
-                "xms_cc": {
-                    "values": ["CP1", "CP2"]
-                }
-            }
-        }
+     *                       <p>
+     *                       Sample input claim :
+     *                       {
+     *                       "userinfo":
```

Comment:
I see the formatter didn't like this inline doc: can we revert the formatter change and wrap the original formatting in a `<pre>` block?

Reply:
This will tell the formatter to leave "as-is" so we can run it in the future and not have to battle with this whitespace change

Reply:
Addressed